backstop/tests/skip-to-content.html

```diff
@@ -177,20 +183,23 @@ <h1>Skip to content link</h1>
           </button>
         </div>
       </header>
-      <div style="border: solid 1px">
-        <p>This is not the main content. Press 'tab' to see skip-to-content-link component. You might need to click on
-          the top of the page or the URL first. Press 'enter' after focusing on skip-to-content-link to skip to the main
-          content below.</p>
+      <br />
```

Comment:
Why changed?

Reply:
I moved the inline styling into `<head>` and the first section didn't really need the box surrounding it.  Also using the `<kbd>` element where we call out the keys within the description.